torrentleech_monitor/settings.py

```diff
@@ -21,7 +21,7 @@
 # General settings.
 
 # If True, E-Mail reports will be sent.
-SHOULD_SEND_REPORT = True
+SHOULD_SEND_REPORT = False
```

Comment:
Why not `True`?

Reply:
Sorry just my setting that i forgot to revert